woocommerce/payment-gateway/External_Checkout/Admin.php

```diff
@@ -288,6 +291,192 @@ protected function get_display_location_options() {
 	}
 
 
+	/**
+	 * Adds admin notices for configuration issues.
+	 *
+	 * @since 5.10.0
+	 */
+	public function add_admin_notices() {
+
+		// if the feature is not enabled, bail
+		if ( ! $this->handler->is_enabled() ) {
+			return;
+		}
+
+		// if not on the settings screen, bail
+		if ( ! $this->is_settings_screen() ) {
+			return;
+		}
+
+		$this->add_configuration_errors_notice();
+
+		$this->add_configuration_warnings_notices();
+	}
+
+
+	/**
+	 * Adds one error notice for all configuration options that need attention.
+	 *
+	 * @since 5.10.0
+	 */
+	protected function add_configuration_errors_notice() {
+
+		$errors = $this->get_configuration_errors();
+
+		if ( ! empty( $errors ) ) {
+
+			$message = sprintf(
+				/* translators: Placeholders:  - external checkout label, %2$s - <strong> tag, %3$s - </strong> tag */
+				__( '%2$S%1$s is disabled.%3$S', 'woocommerce-plugin-framework' ),
+				$this->handler->get_label(),
+				'<strong>', '</strong>',
+			);
+
+			if ( 1 === count( $errors ) ) {
+				$message .= ' ' . current( $errors );
+			} else {
+				$message .= '<ul><li>' . implode( '</li><li>', $errors ) . '</li></ul>';
+			}
+
+			$this->handler->get_plugin()->get_admin_notice_handler()->add_admin_notice( $message, $this->section_id . '-configuration-issue', [
+				'notice_class' => 'error',
+				'dismissible'  => false,
+			] );
+		}
+	}
+
+
+	/**
+	 * Adds warning notices for each configuration option that may need attention.
+	 *
+	 * @since 5.10.0
+	 */
+	protected function add_configuration_warnings_notices() {
+
+		if ( $this->should_display_shipping_based_tax_notice() ) {
+
+			$message = $this->get_shipping_based_tax_notice();
+
+			$this->handler->get_plugin()->get_admin_notice_handler()->add_admin_notice( $message, $this->section_id . '-shipping-based-tax', [
+				'notice_class' => 'warning',
```

Comment:
this needs to be `notice-warning` if it's meant to get the yellow styling

Reply:
Good catch! Tweaked in https://github.com/skyverge/wc-plugin-framework/pull/512/commits/500c052b7f74eef6bf5d821efb9208a77fe2c593.